### PR TITLE
Feature2 added with multiple files

### PR DESCRIPTION
**Pull request by Nimeshworkstation**
i have added geeta and quran

### CODE REVIEW

File: feature2/show.html.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<html>
+<head></head><body>
+
```
Comment (smnighim) [inline feature2/show.html.txt]: change title

File: feature2/hide.html.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<html>
+<head></head><body>
+
+hello hanuman jee, kaise ho aap ?
```
Comment (smnighim) [inline feature2/hide.html.txt]: hanumnam jee must be in capital

**Review by smnighim — CHANGES_REQUESTED** [review]
need necessary actions

File: feature2/show.html.txt (hunk 1 above)
Comment (Nimeshworkstation) [reply feature2/show.html.txt]: i have done change in title

**Pull request closed without merging**